smart_open/s3.py

```diff
@@ -472,6 +473,14 @@ def __exit__(self, exc_type, exc_val, exc_tb):
             self.close()
 
 
+def _insert_spaces(the_string):
```

Comment:
what's a function `_insert_spaces` (here and below)? I see no usages.

Reply:
Used for debugging only. Deleted.